tools/level_scripts/entities.lua

```diff
@@ -117,14 +117,14 @@ sk_definition_writer.add_definition('elevators', 'struct ElevatorDefinition[]',
 local fizzlers = {}
 
 for _, fizzler in pairs(sk_scene.nodes_for_type('@fizzler')) do
-    local position, rotation = fizzler.node.full_transformation:decompose()
+    local position, rotation, scale = fizzler.node.full_transformation:decompose()
```

Comment:
I agree we don't need to go crazy with how the width is calculated, but no other object works this way. It's actually quite simple to take both the scale and mesh x dimension into account:
```lua
local bounding_box = fizzler.node.meshes[1].bb
local width = (bounding_box.max - bounding_box.min).x * scale.x * 0.5
```

This way, the width in Blender will always match the appearance in the game.

Reply:
> It's actually quite simple to take both the scale and mesh x dimension into account:
> 
> ```lua
> local bounding_box = fizzler.node.meshes[1].bb
> local width = (bounding_box.max - bounding_box.min).x * scale.x * 0.5
> ```
> 
> This way, the width in Blender will always match the appearance in the game.

Good to know! I pushed a commit with the adjustment.